### ISSUE TITLE
would this work with numbers instead of words?

### ISSUE THREAD

**Issue by KevinDenys** [paulgb/crossword-composer#5]
Would this work with numbers, if i give it a list of possible numbers it should fit into a grid?

**Comment by paulgb** [comment]
Yes, as long as it’s one digit per grid cell, it should be fine. Just create it with a dictionary that includes numbers only.
